### DISCUSSION TITLE
Feature request: Browse Unihan information

### DISCUSSION THREAD

**Discussion by m13253** [character-map-uwp/Character-Map-UWP#208] [Ideas]
When I use the application to browse a CJK character (e.g. “文”), I hope the corresponding Unihan information can be displayed on the right panel, which is a feature we currently don't have.
![image](https://user-images.githubusercontent.com/1808930/116954877-a0e3b100-ac80-11eb-817f-a86dd64d5012.png)

Unihan database is a database provided by Unicode Consortium that provides pronunciations in Chinese (Mandatin, Cantonese, Tang), Japanese (On, Kun), Korean, and Vietnamese; as well as its meaning in English.
Sometimes there are linked characters (e.g. simplified and traditional versions of the same CJK character) where a list of hyperlinks can be provided for easy navigation.
The latest version is available as a single [Unihan.zip](https://www.unicode.org/charts/unihan.html) in plain text or XML format.

As a comparison, the screenshot of **Gnome Character Map** (aka. gucharmap), which currently only supports Linux:
![image](https://user-images.githubusercontent.com/1808930/116954789-7265d600-ac80-11eb-800b-66ab53500c2a.png)

It would be even better if the contents in the Unihan database can be used for search. (“Search in character details”)
![image](https://user-images.githubusercontent.com/1808930/116955783-0c2e8280-ac83-11eb-8110-977ff99cfd2b.png)

**Comment by JohnnyWestlake** [comment]
Two years later, the Unihan readings database is now integrated. Search may happen at a later point if I understand how it works
